### ISSUE TITLE
Getting Started Checklist: Set site/company name in header/footer checks are bogus

### ISSUE THREAD

**Issue by vicos59** [geodesicsolutions-community/geocore-community#116]
http://localhost:8080/admin/index.php?page=checklist&mc=getting_started

![aaaaaa](https://user-images.githubusercontent.com/6634041/156270437-b8a445b3-6e02-4bb4-ab15-49a2629bc1f0.JPG)

Both just search for a string in the respective header and footer templates. However, since this was written, that data was moved to the database.

Either completely remove the checks, or update the scripts that do the testing to look in the proper location.

src\admin\getting_started_checks\design_headerWebsiteName.php

src\admin\getting_started_checks\design_footerCompanyName.php

**Comment by jonyo** [comment]
@vicos59 By the way, if you did not notice it yet:

Since you are in the `tester` group, I think you should be able to add labels now on issues / PR's.   AKA "triage" the issues.  If you so feel inclined.  That is assuming I got the permissions correct, so if it does not let you, report back so I can fix it if needed.

**Comment by vicos59** [comment]
I was thinking it might be nice to have a tag to prioritize issues. I just did this for my personal project I imported to GitHub.

Be nice to know which ones are priority and need to be fixed before we can release the 1st production release. Some of these are not really showstoppers or even required for the 1st release.

**Comment by vicos59** [comment]
Sounds like milestone is the winner. I need to be able to see what needs to be done to get v20.0 released so I can see where I can help and attack the highest priorities first. Without a goal, we will never get there.
